script/bundle-linux

```diff
@@ -56,7 +56,7 @@ strip --strip-debug "${target_dir}/${target_triple}/release/remote_server"
 
 
 # Ensure that remote_server does not depend on libssl nor libcrypto, as we got rid of these deps.
-ldd "${target_dir}/${target_triple}/release/remote_server" | grep -q 'libcrypto\|libssl'
+! ldd "${target_dir}/${target_triple}/release/remote_server" | grep -q 'libcrypto\|libssl'
```

Comment:
Shellcheck tells me this will silently succeed and not trigger `exit 1` under `set -e`. 😛

Reply:
Uh, that's weird; I recall that I ran this against remote server binary that had these deps and the script did exit? Or maybe I ran just this command itself, not sure.